metis-dereference/metis-dereference-rest/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
+
+services:
+  mongo:
+    image: mongo:4.2.9
+    container_name: metis-dereference-mongo
+    environment:
+      MONGO_INITDB_DATABASE: metis-dereference
+      MONGO_INITDB_ROOT_USERNAME: guest
+      MONGO_INITDB_ROOT_PASSWORD: guest
+    ports:
+      - '27017:27017'
+  metis-dereference-local:
+    image: europeana/metis-dereference:develop
+    container_name: metis-dereference-local
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - '8080:8080'
+    environment:
+      MONGO_HOSTS: metis-core-mongo
```

Comment:
If I understand correctly, we create a container for mongo with name `metis-dereference-mongo` but then within `metis-dereference-local` we set up as a mongo host `metic-core-mongo`, shouldn't we connect to the previously mentioned container `metis-dereference-mongo`?

Reply:
Yes absolutely! I fixed it.